src/power/index.d.ts

```diff
@@ -0,0 +1,32 @@
+import { Multiply, Dec, Cast } from '..';
+
+// Power two numbers: https://lodash.com/docs/4.17.15#power.
+//
+type S = Power<2, 3>; // 8
+//
+// This type uses recursive (and not officially supported) type alias, see more:
+// https://github.com/microsoft/TypeScript/issues/26223#issuecomment-513187373.
+export type Power<
+  // The first number in a multiplication.
+  A extends number,
```

Comment:
I think this can be `B` (short for base).

Reply:
I think it breaks the consistency of other arithmetic computations in the project where A is always the first number of the evaluation (like in normal calculators)